client/verta/docs/tutorials/deployment/endpoint_update_config.rst

```diff
@@ -0,0 +1,63 @@
+Updating endpoint with a configuration file
+===========================================
+
+.. TODO: link to Endpoint.update tutorial
```

Comment:
@nhatsmrt You should be able to link to \`update <endpoint_update.html>`__ now

Reply:
Done.